Gemfile.lock

```diff
@@ -1,9 +1,9 @@
 GIT
   remote: https://github.com/wordpress-mobile/release-toolkit
-  revision: 48f500bd9212f13d2bf71220c89a8794cdd46e03
-  tag: 0.13.0
+  revision: 348de22c625922cca9a5060261fa7a0c4005786e
+  tag: 0.16.0
```

Comment:
0.16.0 is not the latest version. I'll make sure to update to the latest once the full CI automation work is done; I have other gems updates done in my WIP branch.